### PR TITLE
Fixed missing export for Huber loss as tf.keras.losses.huber and tf.keras.metrics.huber

### PR DESCRIPTION
**Pull request by srihari-humbarwadi**

### CODE REVIEW

**Comment by pavithrasv** [comment]
Why is this required? Can we use the Huber loss class instead? The other loss functions are exported because they were available historically.

**Comment by srihari-humbarwadi** [comment]
This would make the API consistent, since all the other loss function are available as a class available and also as a function. And yes we can use Huber loss class just like the rest of the loss classes

File: tensorflow/python/keras/losses.py [hunk 1]
```diff
@@ -1415,6 +1415,7 @@ def categorical_hinge(y_true, y_pred):
   return math_ops.maximum(0., neg - pos + 1.)
 
 
+@keras_export('keras.metrics.huber', 'keras.losses.huber')
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Can we only add it to losses namespace for now? Also you will need to update golden files '.pbtxt' files for API changes.

**Comment by pavithrasv** [comment]
@mihaimaruseac can you advice on how API changes ie.pbtxt files can be updated?

File: tensorflow/python/keras/losses.py (hunk 1 above)
Comment (srihari-humbarwadi) [reply tensorflow/python/keras/losses.py]: Can you please let me know what `.pbtxt` files need changes?

**Review by mihaimaruseac — APPROVED** [review]

**Review by pavithrasv — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
To generate the proto files used for API you need to run the `api_compatibility_test` with `--update_goldens`:

```
bazel build tensorflow/tools/api/tests:api_compatibility_test
bazel-bin/tensorflow/tools/api/tests/api_compatibility_test --update_goldens=True
```

Then, this will need an API review (there are weekly meetings for this) and API approval before being imported

**Comment by alextp** [comment]
As tensorflow/api-owners: we'd rather keep Huber as class-only (as well any future losses).
